### ISSUE TITLE
What parameters?

### ISSUE THREAD

**Issue by amirebrahimi** [Ahdhn/MeshImp#2]
What parameters were used to generate the davidhead level_1.obj and level_2.obj? I used:
```
mesh_imp.exe -sim -sam10 -smooth -dih170 -del -minang -maxang -v davidhead.obj
```

but I get this with holes in it:
![image](https://user-images.githubusercontent.com/904110/64057368-c5ed5b80-cb50-11e9-9cd4-bf9f2b0bd5cc.png)

Also, I'm noticing the triangles are flipped when I import into Unity (see head to the right in the above image), but the original isn't. Is there a way to control that?

**Comment by amirebrahimi** [comment]
I also see this show up the same way in MeshLab FWIW.

**Comment by Ahdhn** [comment]
On clean installation on Windows, I could not reproduce the bug. On [davidhead](https://github.com/Ahdhn/MeshImp/tree/master/data/input/cvt/davidhead) input model and using the input parameters you mentioned above, I can get the following [output](https://github.com/Ahdhn/MeshImp/files/3569759/output.zip)

I believe these holes are due to visual glitch when normal are not consistent. We, unfortunately, did not have normal preservation as one of the objectives we were targeting so we did not pay much attention to it. One easy fix for this is to pass the output obj file to MeshLab and re-orient the normal using `Filters->Normals, Curvatures and Orientation-> Re-Orient all faces coherently`. Applied to the output, I got this [oriented output](https://github.com/Ahdhn/MeshImp/files/3569802/output-oriented.zip).
